packages/grafana-ui/src/components/VizLegend/VizLegendListItem.tsx

```diff
@@ -31,6 +32,32 @@ export const VizLegendListItem: React.FunctionComponent<Props> = ({ item, onSeri
         }}
       />
       <div
+        onMouseMove={(event) => {
+          if (eventBus) {
+            eventBus.publish({
+              type: DataHoverEvent.type,
+              payload: {
+                raw: event,
+                x: 0,
+                y: 0,
+                dataId: item.label,
+              },
+            });
+          }
+        }}
+        onMouseOut={(event) => {
+          if (eventBus) {
+            eventBus.publish({
+              type: DataHoverClearEvent.type,
+              payload: {
+                raw: event,
+                x: 0,
+                y: 0,
+                dataId: item.label,
+              },
+            });
+          }
+        }}
```

Comment:
We should really try to avoid anonymous functions in components. The will be recreated with each re-render.